### ISSUE TITLE
OKX - Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by viniciussvl** [ccxt/ccxt#13513]
Hello, OKX's parseTradingBalance method is returning the error length undefined, more precisely on line 1688 after I have updated to the latest version of CCXT

- OS: Windows 10
- Programming Language version: JavaScript (Node.js)
- CCXT version: 1.84.60

```
if ((eq.length < 1) || (availEq.length < 1)) {
                 account['free'] = this.safeString(balance, 'availBal');
                 account['used'] = this.safeString(balance, 'frozenBal');
             } else {
                 account['total'] = eq;
                 account['free'] = availEq;
             }
```

**Comment by viniciussvl** [comment]
I debugged the two variables eq.length and availEq.length

console.log(eq)
console.log(availEq)

37.04088323357991
undefined

**Comment by kroitor** [comment]
@viniciussvl thanks for reporting it! Fixed in CCXT 1.84.61, the new build will arrive in ~20 minutes. Please, let us know if that does not resolve the issue for you.
